src/components/Button.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import '../index.scss';
+import PropTypes from 'prop-types';
+
+export default class Button extends React.Component {
+  render() {
+    const { name, wide, color } = this.props;
+    const style = {
+      width: wide,
+      background: color,
+    };
+    return (
+      <button type="submit" style={style}>
+        {name}
+      </button>
+    );
+  }
+}
+
+Button.propTypes = {
+  name: PropTypes.string.isRequired,
+  wide: PropTypes.string.isRequired,
+  color: PropTypes.string,
+};
+
```

Comment:
`wide` should be `Boolean` 
https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png

Reply:
Thanks for pointing it out! Is it solved!